test/jest/mock-store.js

```diff
@@ -398,6 +398,11 @@ export const createSwapsMockStore = () => {
               extensionReturnTxHashAsap: false,
             },
           },
+          bsc: {
+            extensionActive: true,
+            mobileActive: false,
+            smartTransactions: {},
```

Comment:
Should the params be inside the `smartTransactions` property?

Reply:
Inside and outside. Outside it's checking if BSC should be enabled in Swaps (with STX on or off) and inside it checks if the network should be enabled for STX.

A long-term solution is to use a new service for feature flags instead of using this endpoint from swaps api.